`melange completion` panics

```shellsession
$ melange completion 
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
chainguard.dev/melange/pkg/cli.New.completion.func2(0xc0001c3d00?, {0x31b9e80?, 0x4?, 0x1d9a73e?})
	chainguard.dev/melange/pkg/cli/completion.go:61 +0x133
github.com/spf13/cobra.(*Command).execute(0xc000588c08, {0x31b9e80, 0x0, 0x0})
	github.com/spf13/cobra@v1.8.1/command.go:989 +0xa91
github.com/spf13/cobra.(*Command).ExecuteC(0xc000588308)
	github.com/spf13/cobra@v1.8.1/command.go:1117 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.8.1/command.go:1041
github.com/spf13/cobra.(*Command).ExecuteContext(0x20728c0?, {0x2072a80?, 0xc000414180?})
	github.com/spf13/cobra@v1.8.1/command.go:1034 +0x47
main.main()
	chainguard.dev/melange/main.go:30 +0x96
```

E.g. `melange completion bash` works though.

--- a/pkg/cli/completion.go
+++ b/pkg/cli/completion.go
@@ -16,7 +16,7 @@ package cli
 
 import (
 	"os"
-	"fmt"
+	"log"
 	"github.com/spf13/cobra"
 )
 
@@ -60,33 +60,32 @@ $ melange completion fish > ~/.config/fish/completions/melange.fish
 		Run: func(cmd *cobra.Command, args []string) {
 			if len(args) == 0 {
 				// Print an error message and exit if no argument is provided
-				fmt.Fprintln(os.Stderr, "Error: A shell type (bash, zsh, fish, powershell) is required.")
-				return
+					log.Fatal("A shell type (bash, zsh, fish, powershell) is required.")
 			}
 		
 			switch args[0] {
 			case "bash":
 				err := cmd.Root().GenBashCompletion(os.Stdout)
 				if err != nil {
-					fmt.Fprintln(os.Stderr, "Error generating Bash completion script:", err)
+					log.Fatalf("Error generating Bash completion script: %v", err)
 				}
 			case "zsh":
 				err := cmd.Root().GenZshCompletion(os.Stdout)
 				if err != nil {
-					fmt.Fprintln(os.Stderr, "Error generating Zsh completion script:", err)
+					log.Fatalf("Error generating Zsh completion script: %v", err)
 				}
 			case "fish":
 				err := cmd.Root().GenFishCompletion(os.Stdout, true)
 				if err != nil {
-					fmt.Fprintln(os.Stderr, "Error generating Fish completion script:", err)
+					log.Fatalf("Error generating fish completion script: %v", err)
 				}
 			case "powershell":
 				err := cmd.Root().GenPowerShellCompletion(os.Stdout)
 				if err != nil {
-					fmt.Fprintln(os.Stderr, "Error generating PowerShell completion script:", err)
+					log.Fatalf("Error generating PowerShell completion script: %v", err)
 				}
 			default:
-				fmt.Fprintln(os.Stderr, "Error: Invalid shell type. Use one of: bash, zsh, fish, powershell.")
+				log.Fatalf("A shell type (bash, zsh, fish, powershell) is required.")
 			}
 		},
 	}

